after.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# The required environment variables are:
+# user - e.g. 'pjmhill'
+
+# optional variables for github commit status API
+# github_token - the token that is used to push commit status
+
+# require to check status
+# result - the result of the jenkins_build.sh -- needs to be captured in Jenkins' cell
+
+
+stack=dev
```

Comment:
ideally, an environment variable like 'stack' would only be defined in one place.  You define it both in 'before' and 'after'